pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1604,6 +1604,26 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vm.Spec, config)
 			Expect(causes).To(BeEmpty())
 		})
+		It("should raise warning when Passt interface API is used", func() {
```

Comment:
In continuation to the prev comment, I would prefer not to see this duplicated for each and every feature we deprecate.